repos/spack_repo/builtin/packages/py_torch/package.py

```diff
@@ -316,8 +321,10 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         depends_on("rocblas")
         depends_on("miopen-hip")
         depends_on("rocminfo")
-        depends_on("aotriton@0.8.1b", when="@2.5:2.6")
-        depends_on("aotriton@0.9.1b", when="@2.7:")
+        depends_on("hipsparselt", when="@2.8:")
+        depends_on("aotriton@0.8b", when="@2.5:")
+        depends_on("aotriton@0.9.2b", when="@2.7")
+        depends_on("aotriton@0.10b", when="@2.8:")
```

Comment:
You are currently saying that PyTorch 2.8 requires aotriton 0.8b and 0.10b, which isn't possible. Do you mean:

```suggestion
        depends_on("aotriton@0.8b", when="@2.5:2.6")
        depends_on("aotriton@0.9.2b", when="@2.7")
        depends_on("aotriton@0.10b", when="@2.8:")
```

Reply:
Yes, 0.8b is for 2.5 and 2.6. Updated this in the recent commit. Thanks